grib/src/main/java/ucar/nc2/grib/coord/EnsCoordValue.java

```diff
@@ -9,10 +9,12 @@
 public class EnsCoordValue implements Comparable<EnsCoordValue> {
   private final int code;
   private final int ensMember;
+  private final int ensNumber;
 
-  public EnsCoordValue(int code, int ensMember) {
+  public EnsCoordValue(int code, int ensMember, int ensNumber) {
```

Comment:
We can't change the signature of a public constructor, but you can add an overload instead

Reply:
Overload done. Adding ensNumber link to the issue #982.